src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java

```diff
@@ -209,4 +212,35 @@ public void testDreamSensitivity() throws Exception {
         // 78
         Assert.assertEquals(snpRecord.getTruePositives() + snpRecord.getFalseNegatives(), 78);
     }
+
+    @Test
+    public void testDoesNotCrashWithNO_VARIATIONAlleles() {
+        final String testDir = toolsTestDir + "concordance/";
+        final File evalVcf = new File(testDir, "noVariationAlleles.vcf");
+        final File truthVcf = new File(testDir, "noVariationAlleles.vcf");
+        final File summary = createTempFile("summary", ".txt");
+
+        final String[] args = {
+                "--" + AbstractConcordanceWalker.EVAL_VARIANTS_LONG_NAME, evalVcf.toString(),
+                "--" + AbstractConcordanceWalker.TRUTH_VARIANTS_LONG_NAME, truthVcf.toString(),
+                "--" + Concordance.SUMMARY_LONG_NAME, summary.toString(),
+        };
+
+        runCommandLine(args);
+
+        try {
+            final ConcordanceSummaryRecord.Reader reader             = new ConcordanceSummaryRecord.Reader(summary);
```

Comment:
There's some strange spacing here.

Reply:
Ah.  I didn't reformat the code before I checked it in.  I like to align my blocks of variable definitions. 

Fixed!